include/envoy/event/scaled_timer.h

```diff
@@ -18,6 +19,10 @@ struct ScaledMinimum {
   inline bool operator==(const ScaledMinimum& other) const {
     return other.scale_factor_.value() == scale_factor_.value();
   }
+  inline friend std::ostream& operator<<(std::ostream& output, const ScaledMinimum& minimum) {
```

Comment:
Please move the definition of all the `operator<<` functions out-of-line (maybe into overload_manager_impl.cc?) to improve compile time.

Reply:
These are used implicitly in both //test/common/event:dispatcher_impl_test and //test/server:overload_manager_impl_test. I had added them to make test failures easier to debug (since googletest will print using operator<< or a PrintTo() override), and I don't know that they're actually used anywhere else. Should I just remove them?

Reply:
If they're unused, then yes, just delete them.

Reply:
Cool, done.